conda_forge_tick/dynamo_models.py

```diff
@@ -0,0 +1,33 @@
+"""The ORM for DynamoDB for the autotick bot"""
+import uuid
+
+from pynamodb.models import Model
+from pynamodb.attributes import (
+    UnicodeAttribute,
+    NumberAttribute,
+)
+
+
+class PRJson(Model):
+    class Meta:
+        table_name = "pr_json"
+        region = "us-east-2"
+
+    id = NumberAttribute(hash_key=True)
+    state = UnicodeAttribute()
+    ETag = UnicodeAttribute(null=True)
+    merged_at = UnicodeAttribute(null=True)
+
+    @classmethod
+    def dump(cls, pr_json: dict):
+        """Dumps a PR JSON object to DynamoDB"""
+        attrs = dict(
+            id=pr_json["id"],
+            state=pr_json["state"],
+            merged_at=pr_json.get("merged_at"),
+            ETag=pr_json.get("ETag"),
+        )
+        if not isinstance(attrs["id"], int):
+            attrs["id"] = -uuid.UUID(attrs["id"]).int
```

Comment:
Why does this line have a minus sign?

Reply:
This is for PRs we don't actually issue

Reply:
Huh?

Reply:
Because they are not valid github ids :(

Reply:
Right, but why are we tracking PRs we did not issue?

Reply:
We didn't issue them because they were done manually before the bot got to them. So we need to know not to issue them. So we pretend that we did issue them so the bot passes it over.

Reply:
Ahhhhhh ok.